Source/WebCore/html/HTMLMediaElement.cpp

```diff
@@ -4984,6 +4985,15 @@ void HTMLMediaElement::mediaPlayerDidRemoveVideoTrack(VideoTrackPrivate& track)
     track.willBeRemoved();
 }
 
+void HTMLMediaElement::mediaPlayerDidReportGPUMemoryFootprint(size_t footPrint)
```

Comment:
Is this function called for increasing memory usage? Or is it reporting current memory footprint? The name implies current usage of memory. If so, adding usage in the following code seems weird.

Reply:
it's reporting current memory footprint